docs/src/report.md

```diff
@@ -81,8 +81,10 @@ Here's a snippet of test results from a Regula report:
 
 **These are the important bits:**
 
-- [Rule Results](#rule-results)
-- [Summary](#summary)
+- [Report Output](#report-output)
```

Comment:
@curtis-fugue I'm inclined to leave the links as they were -- I don't think it necessarily makes sense to link to the top of the page and the "Compliance controls vs. rules" section from here, unless you think there's a good reason to?

Reply:
I don't know why this happened... maybe autoformatting? I'll undo it.